### PR TITLE
Fix missing env in maven docker context generator

### PR DESCRIPTION
**Pull request by TadCordle**
Again, changes for Maven 0.9.13 patch (skipping 0.9.12 to stay even with the Gradle plugin). Will close and re-open with the release PR after approval.

### CODE REVIEW

File: jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java [hunk 1]
```diff
@@ -139,7 +139,7 @@ private static String mapToDockerfileString(Map<String, String> map, String comm
   @Nullable private String baseImage;
   private List<String> entrypoint = Collections.emptyList();
   private List<String> javaArguments = Collections.emptyList();
-  private Map<String, String> environment = Collections.emptyMap();
+  @Nullable private Map<String, String> environment;
```
Comment (coollog) [inline jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java]: Should we have this be consistent with the others and not be nullable?

Comment (TadCordle) [reply jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java]: entrypoint and javaArguments actually become nullable in a future change, I think.

Comment (coollog) [reply jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java]: Ah right. I think if we make environment nullable, we should make exposedPorts and labels nullable too but that change can go in the post-release PR.

**Comment by TadCordle** [comment]
#1153

**Pull request closed without merging**